source/content/guides/decoupled/drupal-backend-starters/05-cache.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Decoupled Drupal Backend Starter for Front-End Sites
+subtitle: Cache Considerations
+description: Learn about caching considerations for your project.
+tags: [webops, workflow, decoupled]
+contributors: [backlineint, cobypear, hckia, whitneymeredith]
+layout: guide
+showtoc: true
+permalink: docs/guides/decoupled/drupal-backend-starters/cache
+anchorid: cache
+contenttype: [guide]
+innav: [true]
+categories: [create]
+cms: [decoupled]
+audience: [development]
+product: [decoupled]
+integration: [--]
+---
+
+This section provides information how to configure caching for your backend project.
+
+You can configure Drupal to cache JSON:API endpoints until the underlying content changes. These same routes can also be cached on a CDN. This can improve performance for Front-End Sites that rely on these API endpoints. This can also reduce the load on your CMS if you make a large number of API requests in a short period of time, such as during a full site build process.
+
+## Use the Backend Starter Project
+
+If you created your Drupal site using the
+[backend starter project](/guides/decoupled/drupal-backend-starters/create), the recommended caching configurations were automatically enabled. Using this project:
+
+- JSON:API is configured to cache responses for 10 minutes.
+- The Pantheon Advanced Page Cache module enables edge caching
+  and purging across the entire decoupled stack.
+
+## Manual Configuration
+
+You must manually configure edge caching and purging if you did not use the Decoupled Kit backend starter for your project.
+
+The edge caching instructions below focus on sites running on Pantheon, but can be adapted for other platforms.
+
+### Enable Caching for JSON:API
+
+1. Open your **Drupal Admin** page, select **Configuration**, select **Development**, and then select **Performance** to access the **Performance settings** page.
+
+1. Set the `Browser and proxy cache maximum age` to a value greater than zero.
+
+#### Enable Edge Caching
+
+1. Run the following Composer command:
+
+  ```bash{promptUser: user}
+  composer require drupal/pantheon_advanced_page_cache
```

Comment:
Shouldn't this module already be in the starter?

Reply:
@CobyPear @backlineint Can you clarify this step?

Reply:
I'm skeptical that "as much information as possible to use Front-End Sites" is the right guiding light here. I think it's more "the right amount of the right information to use Front-End Sites." For someone using our Front-End Sites product, we should not be telling them to run `composer require drupal/pantheon_advanced_page_cache` because that module is already present if they're using the starter provided by Front-End Sites.

I agree with you @whitneymeredith that repeatedly linking to another site in the flow of a single page or series of pages is jarring and adds friction. But this is info that doesn't need to be linked at all (or maybe just linked at the end).

Reply:
@stevector The introductory text at the top states: "Cache settings are configured out-of-the-box for you, but there are circumstances in which you might need to make your own configurations, for example, your CMS is not on Pantheon but your frontend is on Pantheon." We're not telling users this is a required a step.